src/applications/check-in/locales/en/translation.json

```diff
@@ -316,5 +312,18 @@
   "sign-in-to-btsss": "Sign in to BTSSS to check your claim status",
   "we-cant-file-this-type": "We can’t file this type of travel reimbursement claim",
   "were-sorry-we-cant-file-this-type": "We’re sorry. We can’t file this type of travel reimbursement claim for you in this tool right now. But you can still file within 30 days of the appointment through the Beneficiary Travel Self Service System (BTSSS).",
-  "fpo-header": "Placeholder header"
+  "fpo-header": "Placeholder header",
+  "if-you-traveled-by-bus-train-taxi-or-other-authorized--help-text": "<0>If you traveled by bus, train, taxi, or other authorized public transportation, you can’t file a claim in this tool right now.</0> But you can file your claim online, within 30 days, through the Beneficiary Travel Self Service System (BTSSS). Or you can use VA Form 10-3542 to submit a claim by mail, fax, email, or in person.",
+  "if-you-traveled-from-a-different-address-you--helptext": "<0>If you traveled from a different address, you can’t file a claim in this tool right now.</0> But you can file your claim online, within 30 days, through the Beneficiary Travel Self Service System (BTSSS). Or you can use VA Form 10-3542 to submit a claim by mail, fax, email, or in person.",
+  "you-can-submit-your-claim-now-in-this-tool": "You can submit your claim now in this tool or choose to file later.",
+  "if-you-choose-to-file-later": "If you choose to file later, you can text <0>travel</0> to 54804 until the end of today. Or you can file within 30 days through the Beneficiary Travel Self-Service System (BTSSS).",
+  "file-claim": "File claim",
+  "back": "Back",
+  "i-have-incurred-a-cost-in-relation": "I have incurred a cost in relation to the travel claim.",
```

Comment:
this agreement text is already defined in translations under the single key: `certify-statements`, the day-of page uses them. You can output them all in a single Trans component like:
```
<ul>
                <Trans
                  i18nKey="certify-statements"
                  components={[<li key="list-item" />]}
                />
              </ul>
              ```